src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -68,30 +82,49 @@ public JedisSentinelPool(String masterName, Set<String> sentinels,
   public JedisSentinelPool(String masterName, Set<String> sentinels,
       final GenericObjectPoolConfig poolConfig, int timeout, final String password,
       final int database) {
-    this(masterName, sentinels, poolConfig, timeout, timeout, password, database);
+    this(masterName, sentinels, poolConfig, timeout, timeout, password, database, false);
+  }
+
+  public JedisSentinelPool(String masterName, Set<String> sentinels,
+       final GenericObjectPoolConfig poolConfig, int timeout, final String password,
+       final int database, final Boolean isRedisSslEnabled) {
+    this(masterName, sentinels, poolConfig, timeout, timeout, password, database, isRedisSslEnabled);
   }
 
   public JedisSentinelPool(String masterName, Set<String> sentinels,
       final GenericObjectPoolConfig poolConfig, int timeout, final String password,
       final int database, final String clientName) {
-    this(masterName, sentinels, poolConfig, timeout, timeout, password, database, clientName);
+    this(masterName, sentinels, poolConfig, timeout, timeout, password, database, clientName, false);
+  }
+
+  public JedisSentinelPool(String masterName, Set<String> sentinels,
+       final GenericObjectPoolConfig poolConfig, int timeout, final String password,
+       final int database, final String clientName, final Boolean isRedisSslEnabled) {
+    this(masterName, sentinels, poolConfig, timeout, timeout, password, database, clientName, isRedisSslEnabled);
+  }
+
+  public JedisSentinelPool(String masterName, Set<String> sentinels,
+       final GenericObjectPoolConfig poolConfig, final int timeout, final int soTimeout,
+       final String password, final int database) {
+    this(masterName, sentinels, poolConfig, timeout, soTimeout, password, database, null, false);
   }
 
   public JedisSentinelPool(String masterName, Set<String> sentinels,
       final GenericObjectPoolConfig poolConfig, final int timeout, final int soTimeout,
-      final String password, final int database) {
-    this(masterName, sentinels, poolConfig, timeout, soTimeout, password, database, null);
+      final String password, final int database, final Boolean isRedisSslEnabled) {
+    this(masterName, sentinels, poolConfig, timeout, soTimeout, password, database, null, isRedisSslEnabled);
```

Comment:
Don't you think there's a need to add a constructor with `final SSLSocketFactory sslSocketFactory, final SSLParameters sslParameters, final HostnameVerifier hostnameVerifier`

Reply:
Yes. It can be useful. I added this parameters to main constructor of JedisSentinelPool class.

Reply:
@gkorland  What do you think about changes from this https://github.com/xetorthio/jedis/pull/2024/commits/aa02bac03a64ff40ac45657016a5ac4246e063e7 commit?

Reply:
:+1: